requirements/base.in

```diff
@@ -9,6 +10,5 @@ ratelimit
 requests
 pypdf
 pycryptodome
-# Note(austin) higher versions need rust installed? pinning this until we address in the dockerfile
-safetensors<0.3.3
```

Comment:
this should probably still be pinned consistent with:
https://github.com/Unstructured-IO/unstructured/blob/main/requirements/constraints.in

Reply:
oh, i see it moved. nvm.